test/bootstrap.jl

```diff
@@ -184,7 +184,12 @@ end
         @test pb1 == restorereplicates(seekstart(io), MixedModels.unfit!(deepcopy(m1)))
 
         @test pb1 ≈ restorereplicates(seekstart(io), m1, Float16) rtol=1
-    end
+        # these tests must be last because the tbl extractor changes the λ field in the sample
```

Comment:
Why does the extractor change the lambda field? Are we using the lambda field anywhere else where that change might lead to unexpected behavior depending on the order a user looks at the `tbl` property?

Reply:
The way that the standard deviation and correlation parameters are evaluate from sigma and theta is to populate lambda with the values of the theta parameters and determine sigma times the length of each row in each element of lambda.  While doing this for a lower triangular component of lambda I normalize the row after taking its length then evaluate the inner products of the normalized rows, which gives the correlations.  If we were confident that the lambda value always corresponds to the last value of theta, which is what happens when it is generated, we could simply install that value of theta.  Otherwise we should extract theta from lambda, create the table and reinstall the saved theta.  It means we need to have get\theta code as well as set\theta!   Alternatively, and more simply, we could save a copy of lambda, create the table and copy the saved lambda back into the bootstrap struct.

Reply:
> Alternatively, and more simply, we could save a copy of lambda, create the table and copy the saved lambda back into the bootstrap struct.

I think this is the way to go.